randovania/games/am2r/json_data/Main Caves.txt

```diff
@@ -817,6 +819,8 @@ Extra - minimap_data: [{'x': 46, 'y': 37}, {'x': 46, 'y': 38}, {'x': 46, 'y': 39
               Space Jump or Walljump (Hypermode) or Can Use Spider Ball
               # IBJ
               Infinite Bomb Jumping (Expert) and Can Use Bombs
+          # Shinespark
+          Speed Booster and Shinesparking Tricks (Beginner) and Disabled Entrance Rando
```

Comment:
I'm surprised this was never logical lmao

Is this viable with extreme stuttering?

Reply:
I do not have the skill to confirm but seems like one probably could do it. Will need someone to test and confirm a Short Charge difficulty.
Sidenote: Add HJ+WJ (Adv)

Reply:
> Will need someone to test and confirm a Short Charge difficulty.

Add a todo then, and call it a day

> Sidenote: Add HJ+WJ (Adv)

good catch.